LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -912,6 +998,9 @@ $(function() {
     	}
     });
 
+
```

Comment:
Be cautious adding a bunch of whitespace lines.  Lint will be whining about them.